src/components/ContactDetails.vue

```diff
@@ -485,17 +490,18 @@ export default {
 
 		/**
 		 * Store getters filtered and mapped to usable object
-		 * This is the list of addressbooks that are available to write
+		 * This is the list of addressbooks that are available
 		 *
-		 * @return {{id: string, name: string}[]}
+		 * @return {{id: string, name: string, readOnly: boolean}[]}
 		 */
 		addressbooksOptions() {
 			return this.addressbooks
-				.filter(addressbook => !addressbook.readOnly && addressbook.enabled)
+				.filter(addressbook => addressbook.enabled)
 				.map(addressbook => {
 					return {
 						id: addressbook.id,
 						name: addressbook.displayName,
+						readOnly: addressbook.readOnly,
 					}
 				})
```

Comment:
This could have side effects. However, I did not spot any during my testing and the address book options made sense.